### DISCUSSION TITLE
Is anyone using this library in production?

### DISCUSSION THREAD

**Discussion by tyleralbee** [TanStack/router#742] [General]
I understand that this project is in beta, changes often, and that there are issues. I am more than aware it is not a good idea to use this library in production. However, that didn't stop me from integrating it into a project I started building back in May, as I loved the idea behind it & there is simply no alternative. My project's MVP deadline is Oct 1, and I worry that 6 months from that date that it still won't be ready for production use.

I would bet that there's others in my position, and I am wondering: if you are using this library in production, which version are you using?

**Comment by hosmelq** [comment]
Hello @tyleralbee,

I'm using it for https://plutolinks.com (close beta). There have been times when I have had to refactor due to a breaking change and at other times rollback due to a bug. but after that everything was amazing. At the moment I'm at 0.0.1-beta.191.
